src/types/save-svg-as-png.d.ts

```diff
@@ -0,0 +1,39 @@
+// Type definitions for saveSvgAsPng v1.0.3
+// Project: https://github.com/exupero/saveSvgAsPng
+
+declare module "save-svg-as-png" {
+    export type SourceElement = HTMLElement | SVGElement | Element
+
+    export type BackgroundStyle = string | CanvasGradient | CanvasPattern
+
+    export interface SelectorRemap {
+        (text: string): string
+    }
+
+    export interface SaveSVGOptions {
+        scale?: number
+        responsive?: boolean
+        width?: number
+        height?: number
+        left?: number
+        top?: number
+        selectorRemap?: SelectorRemap
+        backgroundColor?: BackgroundStyle
+    }
+
+    export interface UriCallback {
+        (uri: string): void
+    }
+
+    export function svgAsDataUri(el: SourceElement, options: SaveSVGOptions, cb: UriCallback): void
+
+    export function svgAsPngUri(el: SourceElement, options: SaveSVGOptions, cb: UriCallback): void
+
+    export function saveSvg(el: SourceElement, fileName: string, options?: SaveSVGOptions): void
+
+    export function saveSvgAsPng(
+        el: SourceElement,
+        fileName: string,
+        options?: SaveSVGOptions,
+    ): void
+}
```

Comment:
Installing `types-save-svg-as-png` instead of `@types/save-svg-as-png` might work.
https://www.npmjs.com/package/types-save-svg-as-png

Maybe we should use a different library for rendering the SVG to PNG as this one is no longer maintained?
https://github.com/canvg/canvg looks promising.

Actually, we might not need a library for this. One can draw SVG images to Canvas using the Image and Blob API.
https://levelup.gitconnected.com/draw-an-svg-to-canvas-and-download-it-as-image-in-javascript-f7f7713cf81f

Reply:
Implemented canvas+blob download in https://github.com/lgfrbcsgo/du-factory-generator/commit/9b42f51221614e7eb0d97414c04c1a96d246c4ee